HarpGuru/components/index.tsx

```diff
@@ -30,89 +15,18 @@ import {
   QuizQuestionScreen,
 } from '../Screens'
 
-setGlobalReactNState()
+import { useSwipeMenus, MenuStates } from './useSwipeMenus'
+import { useQuizCycle } from './useQuizCycle'
```

Comment:
I would like to see if the entire quiz cycle can be managed from within this one hook. Doing so would improve readibility, but it may also have performance improvements as described in https://github.com/js-jslog/harpnative/issues/52#issuecomment-663847598

This should wait for the next pull request though.